docs/control-structures.rst

```diff
@@ -206,7 +206,7 @@ Those names will still be present on the stack, but they are inaccessible.
             return k;
         }
     }
-    
+
```

Comment:
Please don't do unneeded formatting changes.

Reply:
I didn't :) 
There was an extra space on that line. Since the `.editorconfig` has `trim_trailing_whitespace` set to `true`, it removed the space when I saved the file. Guess the person working on the file last didn't realise.